react/personal_project_vite/src/index.css

```diff
@@ -63,7 +76,9 @@ li {
 .listing{
   display: flex;
   justify-content: space-around;
-  margin: 5px;
+  margin: 10px;
+  border-top: 1px dashed rgb(162, 48, 162);
+  padding-top: 10px;
 }
```

Comment:
There seems to be a mix of responsive and none responsive units of measurement. It's likely not a priority for now but moving forward stick to one or the other to ensure consistency with your programs display.